component/class/defaults.yml

```diff
@@ -457,7 +457,7 @@ parameters:
                 memory: "30Mi"
               limits:
                 cpu: "32m"
-                memory: "2Gi"
+                memory: "64Mi"
```

Comment:
This actually might be too low. Have you verified that it works with this?

Reply:
I agree, I would go with 256 or something.